kafka_utils/util/offsets.py

```diff
@@ -342,18 +342,18 @@ def _commit_offsets_to_watermark(
 
     if watermark == HIGH_WATERMARK:
         group_offset_reqs = [
-            OffsetCommitRequest(
-                kafka_bytestring(topic), partition,
-                watermark_offsets[topic][partition].highmark, None
+            OffsetCommitRequestPayload(
+                topic, partition,
+                watermark_offsets[topic][partition].highmark, ''
             )
             for topic, partitions in topics.iteritems()
             for partition in partitions
         ]
     elif watermark == LOW_WATERMARK:
         group_offset_reqs = [
-            OffsetCommitRequest(
-                kafka_bytestring(topic), partition,
-                watermark_offsets[topic][partition].lowmark, None
+            OffsetCommitRequestPayload(
+                topic, partition,
+                watermark_offsets[topic][partition].lowmark, ''
```

Comment:
could we say metadata='', is more descriptive and helps not to jump around in code